torch/package/_file_structure_representation.py

```diff
@@ -4,27 +4,47 @@
 from .glob_group import GlobPattern, GlobGroup
 
 
-class Folder:
+class Directory:
+    """A file structure representation. Organized as Directory nodes who have lists of
```

Comment:
```suggestion
    """A file structure representation. Organized as Directory nodes that have lists of
```

Reply:
Also, consider using 
```
:class:`Directory`
```
instead of just `Directory`. This will render in the API reference as a link to the `Directory` class. But maybe it's not useful to have that link because this *is* the `Directory` class; it makes more sense when the docstring refers to other classes.

Reply:
I used the `:class:'Directory'` string in the other public API to enable linking 👍